### PR TITLE
[FLINK-17764][Examples]Update tips about the default planner when the  planner parameter value is not recognized

### PR DESCRIPTION
**Pull request by Shih-Wei-Hsu**
## What is the purpose of the change

Keep the default planner in the prompt consistent with the code.

## Brief change log

Change tips about the default planner to blink when the planner parameter value is not recognized , because this default planner has been set to blink in the code.

## Verifying this change

trivial change

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by Shih-Wei-Hsu** [comment]
Hi @dianfu ,

First of all thank you for assigning it to me. 
Now this PR Test seems to have been successful. 
Can you help me review it, and merge it?

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public static void main(String[] args) throws Exception {
 			tEnv = StreamTableEnvironment.create(env, settings);
 		} else {
 			System.err.println("The planner is incorrect. Please run 'StreamSQLExample --planner <planner>', " +
-				"where planner (it is either flink or blink, and the default is flink) indicates whether the " +
```
Comment (dianfu) [inline flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java:72]: Could you also update the class StreamSQLExample.scala?

**Comment by dianfu** [comment]
@Shih-Wei-Hsu Thanks a lot for PR. LGTM with just one minor comments. Will update it when merging the PR.

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java (hunk 1 above)
Comment (Shih-Wei-Hsu) [reply flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java:72]: Thanks for your reminder.
I actually ignored it. 
I will submit this update later.

**Pull request closed without merging**